src/sentry/integrations/custom_scm/integration.py

```diff
@@ -52,6 +52,19 @@ class CustomSCMIntegration(IntegrationInstallation, RepositoryMixin):
     def get_client(self):
         pass
 
+    def get_stacktrace_link(self, repo, filepath, default, version):
```

Comment:
adding in this stuff since it's low effort, all we are doing is formatting a string